great_expectations/validator/validator.py

```diff
@@ -1176,10 +1176,13 @@ def _resolve_suite_level_graph_and_process_metric_evaluation_errors(
             Tuple[str, str, str],
             Dict[str, Union[MetricConfiguration, Set[ExceptionInfo], int]],
         ]
-        resolved_metrics, aborted_metrics_info = graph.resolve(
+        (
+            resolved_metrics,
+            aborted_metrics_info,
+        ) = self.metrics_calculator.resolve_validation_graph(
```

Comment:
Hi @alexsherstinsky  is `MetricsCalculator` a recent development? Do we intend all of the `resolve()` steps to use it instead of the `Validator` doing it directly?

Reply:
@Shinnnyshinshin It has been around since early October, 2022.  In fact the `Validator` now proxies to `MetricsCalculator` for these steps (for backward compatibility).  Thanks.

Reply:
Oo ok. Could you tell me a little bit more about the backwards compatibility?

Reply:
From conversation : backwards compatibility for `Validator`.